specs/_features/eip7594/polynomial-commitments-sampling.md

```diff
@@ -180,6 +179,40 @@ def fft_field(vals: Sequence[BLSFieldElement],
         return _fft_field(vals, roots_of_unity)
 ```
 
+#### `coset_fft_field`
+
+```python
+def coset_fft_field(vals: Sequence[BLSFieldElement], 
+                    roots_of_unity: Sequence[BLSFieldElement], 
```

Comment:
A bit of trailing whitespace here and along this function in general. Also in the docs of construct_vanishing_polynomial(). Maybe you should ask your editor to highlight them?

Reply:
Can you elaborate on the issue here? the linter is not showing up any issues

Reply:
Just an extra space at the end. The linter won't complain about trailing whitespace.

<img width="511" alt="image" src="https://github.com/ethereum/consensus-specs/assets/95511699/6b990c7a-56bb-4828-9200-dc01f5e17e9b">

<img width="511" alt="image" src="https://github.com/ethereum/consensus-specs/assets/95511699/feae1b51-4a6c-4016-96e6-4dd207413b9f">

Reply:
Here's how you might auto-remove trailing whitespace in vscode:

* https://stackoverflow.com/a/30884298

Reply:
Gracias